community.md

```diff
@@ -27,10 +27,11 @@ Below is a list of projects and resources from the Blazor community (links in ea
 
 ## Libraries
 
-* [BlazorDB](https://github.com/chanan/BlazorDB) &ndash; In-memory, persisted to local storage, database for Blazor.
+* [BlazorBits](http://blazorbits.net) ([GitHub repo](https://github.com/blazorbits)) &ndash; Components including [Monaco editor](https://github.com/Microsoft/monaco-editor).
```

Comment:
Is blazorbits.net still live? The link isnt working for me. The repo link is fine.

Reply:
Interesting - It's working fine for me. Make sure you're on http and not https as it's a custom domain on github pages at the moment?

Reply:
Its working for me now. Probably was just a network issue on my end.